opentelemetry-api/src/opentelemetry/environment_variables/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OTEL_PROPAGATORS = "OTEL_PROPAGATORS"
+OTEL_PYTHON_CONTEXT = "OTEL_PYTHON_CONTEXT"
+OTEL_PYTHON_DISABLED_INSTRUMENTATIONS = "OTEL_PYTHON_DISABLED_INSTRUMENTATIONS"
+OTEL_PYTHON_IDS_GENERATOR = "OTEL_PYTHON_IDS_GENERATOR"
+OTEL_PYTHON_SERVICE_NAME = "OTEL_PYTHON_SERVICE_NAME"
+OTEL_TRACE_EXPORTER = "OTEL_TRACE_EXPORTER"
```

Comment:
is there a reason to have the exporter vars in both the sdk and the api?

Reply:
Fixed